raiden/transfer/channel.py

```diff
@@ -133,6 +134,16 @@ class UnlockGain(NamedTuple):
     from_partner_locks: TokenAmount
 
 
+class IsChannelUsable(Enum):
```

Comment:
This sounds to me like a boolean, what about `ChannelUsability` and renaming `YES` to `USABLE`?